js/main.js

```diff
@@ -2,3 +2,11 @@
 if (typeof feather !== "undefined") {
   feather.replace();
 }
+
+var links = document.links;
+
+for (var i = 0, linksLength = links.length; i < linksLength; i++) {
```

Comment:
Actually is it possible to limit these changes to the Ksphere docs? Other docs and teams might not want this behavior.

Reply:
@cmastr - What do you think? ☝️

Reply:
To follow up - Casie said let's make it universal.